sdks/python/apache_beam/utils/proto_utils_test.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from google.protobuf import timestamp_pb2
+
+from apache_beam.utils.proto_utils import pack_Any
+from apache_beam.utils.proto_utils import to_Timestamp
+from apache_beam.utils.proto_utils import unpack_Any
+
+
+class ProtoUtilsTest(unittest.TestCase):
+  def make_proto_timestamp(self):
+    # type: () -> timestamp_pb2.Timestamp
+    return to_Timestamp(0)
+
+  def test_none_pack(self):
+    packed_none = pack_Any(None)
+    assert packed_none is None
+
+  def test_date_pack(self):
+    # type: () -> None
+    proto_timestamp = self.make_proto_timestamp()
+    packed_msg = pack_Any(proto_timestamp)
+    orig_msg = unpack_Any(packed_msg, timestamp_pb2.Timestamp)
+    none_msg = unpack_Any(packed_msg, None)
+    assert proto_timestamp == orig_msg
+    assert none_msg is None
```

Comment:
is this a left over given that you have a separate test_none_pack?

Reply:
test_None_pack is to check packing None. unpack_Any does not allow None for the msg, but allows None for the msg type. So this is intended. I wanted to test both None msg pack and None class unpack.